frontend/src/pages/ChainEdit.js

```diff
@@ -149,33 +149,83 @@ const ChainEdit = () => {
                 }
               />
 
-              <div style={{ display: "flex" }}>
+              <div style={{ display: "flex", flexDirection: "column" }}>
                 <div>
-                  <Typography component="p">categories</Typography>
-                  {categories.genders.map((value) => {
-                    return (
-                      <CheckboxField
-                        name={value}
-                        label={`${value}'s clothing`}
-                        onChange={(e) => handleChange(e, genders, setGenders)}
-                        checked={genders[value] ? true : false}
-                      />
-                    );
-                  })}
+                  <Typography
+                    component="p"
+                    style={{ textAlign: "left", textTransform: "capitalize" }}
+                  >
+                    categories
+                  </Typography>
+                  <div style={{ display: "flex", padding: "2% 0" }}>
+                    {categories.genders.map((value, i) => {
+                      return (
+                        <div key={value}>
+                          <input
+                            className="map-cat-input"
+                            key={`input-${value}-${i}`}
+                            id={`${value}`}
+                            type="checkbox"
+                            name={value}
+                            onChange={(e) =>
+                              handleChange(e, genders, setGenders)
+                            }
+                            checked={genders[value] ? true : false}
```

Comment:
Isn't this just `checked={genders[value]}`

genders is already a boolean array. Otherwise try `checked={!!genders[value]}` (https://www.samanthaming.com/tidbits/19-2-ways-to-convert-to-boolean/).